.github/workflows/docker.yml

```diff
@@ -19,73 +19,9 @@ jobs:
         uses: styfle/cancel-workflow-action@0.8.0
         with:
           access_token: ${{ github.token }}
-
-  build_deb:
```

Comment:
The node cli does not download the debian package but the other build artifacts.
Those two for example:
- babylon-node-rcnet-v1-c6360105d.zip
- babylon-node-rust-arch-linux-x86_64-release-rcnet-v1-c6360105d.zip

We actually build all the parts for the systemd run ourselves. (like the systemd config file and dependencies)
I agree however that building a working debian package can be useful and should be published. But atm, I have a strong hunch that its not working smoothly. I would re-add it at a later stage, or just keep it now and not use the artifact. 

Here I just wanted to safe space and build time.

leave me a thumbs up for adding it back in or a thumbs down for keeping it deleted.

Reply:
I’m happy for us to ditch it for this PR. But could I ask you to follow up with it later?

We may even want to ditch it for good. It might be worth us asking node runners if anyone’s using it.

If we choose to ditch it, we should ditch all the gradle stuff for building the image.

If we choose to keep it, we should probably test it somehow.

On a separate note I think the deb build might be responsible for giving the build a name which somehow gets onto the top-right of the dashboard - so we should probably check that the build number / tag is included via some other means in this build :)